show/main.py

```diff
@@ -2082,6 +2082,134 @@ def ztp(status, verbose):
     run_command(cmd, display_cmd=verbose)
 
 
+#
+# 'bmp' group ("show bmp ...")
+#
+@cli.group(cls=clicommon.AliasedGroup)
+def bmp():
+    """Show details of the bmp dataset"""
+    pass
+
+
+# 'bgp-neighbor-table' subcommand ("show bmp bgp-neighbor-table")
+@bmp.command('bgp-neighbor-table')
+@clicommon.pass_db
+def bmp_neighbor_table(db):
+    """Show bmp bgp-neighbor-table information"""
+    bmp_headers = ["Neighbor_Address", "Peer_Address", "Peer_ASN", "Peer_RD", "Peer_Port",
+                   "Local_Address", "Local_ASN", "Local_Port", "Advertised_Capabilities", "Received_Capabilities"]
+
+    # BGP_NEIGHBOR_TABLE|10.0.1.2
+    bmp_keys = db.db.keys(db.db.STATE_DB, "BGP_NEIGHBOR_TABLE|*")
```

Comment:
>STATE_DB [](http://example.com/codeflow?start=32&length=8)

As offline discussed, please use a dedicated database for BMP datasets, proposing the database name is BMP_STATE_DB. you can pick one unused database index [0, 100). 100 is current maximal.

Reply:
done

Reply:
you are still using STATE_DB, as commented in other PR, your will have performance issue due to large amount of BMP entries

Reply:
seems you're checking old revision, could you double check the latest revision? thanks